sdks/java/extensions/google-cloud-platform-core/src/test/java/org/apache/beam/sdk/extensions/gcp/util/GcsUtilTest.java

```diff
@@ -497,21 +497,30 @@ public void testGetSizeBytesWhenFileNotFoundBatchRetry() throws Exception {
             + "\n";
     thrown.expect(FileNotFoundException.class);
 
-    final LowLevelHttpResponse mockResponse = Mockito.mock(LowLevelHttpResponse.class);
-    when(mockResponse.getContentType()).thenReturn("multipart/mixed; boundary=" + contentBoundary);
+    final LowLevelHttpResponse[] mockResponses =
+        new LowLevelHttpResponse[] {
+          Mockito.mock(LowLevelHttpResponse.class), Mockito.mock(LowLevelHttpResponse.class),
```

Comment:
2 mock objects for 2 responses.

Reply:
And today I came to know @kennknowles knew this 18 days ago already.

https://github.com/apache/beam/pull/14218#issuecomment-812091162